examples/virus_on_network/app.py

```diff
@@ -1,5 +1,4 @@
 import math
-
 import solara
```

Comment:
Our [ruff](https://github.com/astral-sh/ruff) formatter wants the empty line here: it separates the math import (a Python built-in) from the other imported packages.